README.md

```diff
@@ -16,6 +16,15 @@ CLIENT_REPO_ROOT=${PWD}/c
 # Install pre-requisites
 sudo apt-get install libssl-dev libcurl4-openssl-dev uncrustify libyaml-dev
 
+# Build pre-requisite: libwebsockets
+git clone https://libwebsockets.org/repo/libwebsockets
```

Comment:
We should tag at a specific commit not HEAD

Reply:
Done.